docs/sphinx/sections/api/apidocs/libraries/dagster-airbyte.rst

```diff
@@ -0,0 +1,21 @@
+Airbyte (dagster-airbyte)
+---------------------------
+
+This library provides a Dagster integration with `Airbyte <https://www.airbyte.com/>`_.
+
+.. currentmodule:: dagster_airbyte
+
+Ops
+===
+
+.. autofunction:: airbyte_sync_op
+
+
+Resources
+=========
+
+.. autodata:: airbyte_resource
```

Comment:
this (and `airbyte_sync_op`) can be `autoconfigurable` (that's a custom sphinx extension we have that will document the config schema of things).